### PR TITLE
docs: adds source compatibility note

### PR DESCRIPTION
**Pull request by heyitsbryanm**
## Description

Some customers don't know what Sources are compatible:
https://imgix.slack.com/archives/CHW1PHD47/p1673282755361489?thread_ts=1673282100.211479&cid=CHW1PHD47

I also don't want to continue updating the README whenever we update a Source, so:

- I specified what is _not_ supported
- I added a link to our documentation

## Checklist

- [ ] Read the [contributing guidelines](CONTRIBUTING.md).
- [ ] Each commit follows the [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) spec format.
- [ ] Update the readme (if applicable).
- [ ] Update or add any necessary API documentation (if applicable)
- [ ] All existing unit tests are still passing (if applicable).

- [ ] Add some [steps](#steps-to-test) so we can test your bug fix or feature (if applicable).
- [ ] Add new passing unit tests to cover the code introduced by your PR (if applicable).
- [ ] Any breaking changes are specified on the commit on which they are introduced with `BREAKING CHANGE` in the body of the commit.
- [ ] If this is a big feature with breaking changes, consider opening an issue to discuss first. This is completely up to you, but please keep in mind that your PR might not be accepted.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +63,10 @@ https://user-images.githubusercontent.com/15919091/137049820-8ffc4bfd-43f1-41d2-
   <source src="https://user-images.githubusercontent.com/15919091/137049820-8ffc4bfd-43f1-41d2-b078-068ddaaa0c86.mp4">
 </video>
 
+### Compatibility
+
+This Contentful app works is **not compatible with Web Folder or Web Proxy Sources**. [See the imgix documentation](https://docs.imgix.com/setup/creating-sources) for a list of Sources that you can create in imgix.
```
Comment (luqven) [inline README.md:68]:
```suggestion
This Contentful app is **not compatible with Web Folder or Web Proxy Sources**. [See the imgix documentation](https://docs.imgix.com/setup/creating-sources) for a list of Sources that you can create in imgix.
```

**Review by luqven — CHANGES_REQUESTED** [review]
LGTM let's just fix that grammar error, and we can get this merged in.

File: README.md (hunk 1 above)
Comment (sherwinski) [reply README.md:68]: Just double checking but don't we technically support web folder sources? I recall that users just can't upload to a WF.

Comment (luqven) [reply README.md:68]: Good catch, yes. I read this too quickly and didn't notice it didn't say incompatible _with upload_.

Comment (heyitsbryanm) [reply README.md]: Thanks for catching those. The changes were committed, feel free to merge.

File: README.md [hunk 2]
```diff
@@ -63,6 +63,10 @@ https://user-images.githubusercontent.com/15919091/137049820-8ffc4bfd-43f1-41d2-
   <source src="https://user-images.githubusercontent.com/15919091/137049820-8ffc4bfd-43f1-41d2-b078-068ddaaa0c86.mp4">
 </video>
 
+### Compatibility
+
+This Contentful app is **not compatible with Web Proxy Sources**. Uploading functionality is not **not compatible** with Web Folder sources. [See the imgix documentation](https://docs.imgix.com/setup/creating-sources) for a list of Sources that you can create in imgix.
```
Comment (sherwinski) [inline README.md:68]: Sorry, just one last thing
```suggestion
This Contentful app is **not compatible with Web Proxy Sources**. Uploading functionality is **not compatible** with Web Folder sources. [See the imgix documentation](https://docs.imgix.com/setup/creating-sources) for a list of Sources that you can create in imgix.
```

**Review by sherwinski — APPROVED** [review]

**Pull request merged**